cocos/3d/framework/model-component.js

```diff
@@ -87,7 +87,6 @@ let ModelShadowCastingMode = Enum({
  */
 @ccclass('cc.ModelComponent')
 @executionOrder(100)
-@executeInEditMode
```

Comment:
It seems if we do not use `@executeInEditMode`, then `onEnable` `onDisable` will not be called in editor.  model will show wrongly. Am I right @jareguo ?